components/casemap/tests/gen_greek_to_me.rs

```diff
@@ -170,9 +168,10 @@ pub(crate) fn match_extras(ch: char) -> Option<u8> {{
         vec_1f00.len(),
     );
 
-    let local = Path::new(std::env!("CARGO_MANIFEST_DIR")).join("src/greek_to_me/data.rs");
-
-    let local = fs::read_to_string(local).expect("src/greek_to_me/data.rs should be a UTF-8 file");
+    let local = include_str!(concat!(
```

Comment:
`include` is always resolved relative to the source file, so there's no need to involve `CARGO_MANIFEST_DIR`

also everywhere below